resources/default.rb

```diff
@@ -0,0 +1,133 @@
+property :create_user, [true, false], default: true
+property :flyway_user, String, default: 'flyway'
+property :flyway_group, String, default: 'flyway'
+property :install_java, [TrueClass, FalseClass], default: true
+property :version, String, default: '3.2.1'
+property :flyway_url, String
+property :install_path, String, default: lazy { platform_family?('windows') ? 'C:\\flyway' : '/opt/flyway' }
+property :jdbc_driver, String
+property :migrations_path, String, default: lazy { platform_family?('windows') ? 'C:\\flyway\\sql' : '/opt/flyway/sql' }
+property :migrations_url, String
+property :migrations_strip_components, Integer, default: 0
+property :db_url, String
+property :db_user, String
+property :db_password, String
+property :options, Hash
+
+action_class do
+  def new_flyway_url
+    'http://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/VERSION/flyway-commandline-VERSION.tar.gz'
```

Comment:
better keep those URLs as external attributes, so that's simpler to change them without changing the cookbook